api/src/test/java/org/apache/iceberg/expressions/TestTimestampLiteralConversions.java

```diff
@@ -107,6 +108,33 @@ public void testTimestampMicrosToDateConversion() {
     assertThat(dateOrdinal).isEqualTo(-1);
   }
 
+  @Test
+  public void testTimestampNanoWithLongLiteral() {
+    // verify round-trip between timestamp_ns and long
+    Literal<Long> timestampNano =
+        Literal.of("2017-11-16T14:31:08.000000001").to(Types.TimestampNanoType.withoutZone());
```

Comment:
nit: `.123456789` is easier to read that there are 9 digits for nano

I created a PR for this along with changes on naming. `timestampNano` and `longLiteral` doesn't fully capture what they represent.
https://github.com/ebyhr/iceberg/pull/1

Reply:
Thanks for your suggestion, updated.